### PR TITLE
Message Box Component Added

### PR DESCRIPTION
**Pull request by mvhk**
Used [Alerts](https://tailwindcss.com/components/alerts/#traditional)

### CODE REVIEW

File: src/uielements/message/Readme.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+Simple Message example:
+
+```js
```
Comment (udaypydi) [inline src/uielements/message/Readme.md:3]: message should not have an empty messageText Prop. Make it required.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (udaypydi) [inline .gitignore:1]: why is this file changed?

File: src/app.jsx [hunk 3]
```diff
@@ -10,17 +10,16 @@ export default function App() {
 
   return (
     <div className="bg-blue-900 flex flex-col justify-center items-center p-20">
-      <p className="text-4xl text-blue-100">Welcom To Tail UI!</p>
-      <p className="text-2xl text-blue-100">React UI Components Built Using TailwindCSS!</p>
+      <p className="text-4xl text-blue-100">Welcome To Tail UI!</p>
```
Comment (udaypydi) [inline src/app.jsx:13]: revert this file to master

File: src/docs/button/index.jsx [hunk 4]
```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import Button from 'uielements/button/button.component';
-
+import Message from 'uielements/message/message.component';
```
Comment (udaypydi) [inline src/docs/button/index.jsx:3]: why is there a message component in this. Revert this file to default

File: src/uielements/button/button.component.js [hunk 5]
```diff
@@ -7,7 +7,14 @@ import 'src/assets/css/main.css';
 
 function Button(props) {
   const {
-    buttonText, primary, onClick, className, style, bgColor, color, rounded,
+    buttonText,
```
Comment (udaypydi) [inline src/uielements/button/button.component.js:10]: pull master and disable auto indentation in your editor.

File: src/uielements/input/input.component.js [hunk 6]
```diff
@@ -4,40 +4,44 @@ import classnames from 'classnames';
 
 function Input(props) {
   const {
-    placeholder, className, rounded, disabled, error, fluid, style, type, onChange,
+    placeholder,
```
Comment (udaypydi) [inline src/uielements/input/input.component.js:7]: pull master and disable auto indentation in your editor.

File: src/uielements/message/message.component.js [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+function Message(props) {
+  const { className, messageText, color, style, rounded, primary } = props;
+  let messageClass;
+  if (primary) {
+    messageClass = `bg-${color}-100 border border-${color}-400 text-${color}-700 px-4 py-3 relative`;
+  }
+
+  if (rounded) {
+    messageClass += ' rounded-md';
```
Comment (udaypydi) [inline src/uielements/message/message.component.js:12]: use classnames. like the way it is used in other components

File: src/uielements/message/message.component.js [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+  }
+
+  return (
+    <div className={`${messageClass} ${className}`} style={style}>
```
Comment (udaypydi) [inline src/uielements/message/message.component.js:16]: `classnames` library

File: src/uielements/message/message.component.js [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+      <span className="block sm:inline"> {messageText}</span>
+      <span className="absolute top-0 bottom-0 right-0 px-4 py-3">
+        <svg
+          className="fill-current h-6 w-6 text-red-500"
+          role="button"
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 20 20"
+        >
+          <title>Close</title>
+          <path d="M14.348 14.849a1.2 1.2 0 0 1-1.697 0L10 11.819l-2.651 3.029a1.2 1.2 0 1 1-1.697-1.697l2.758-3.15-2.759-3.152a1.2 1.2 0 1 1 1.697-1.697L10 8.183l2.651-3.031a1.2 1.2 0 1 1 1.697 1.697l-2.758 3.152 2.758 3.15a1.2 1.2 0 0 1 0 1.698z" />
```
Comment (udaypydi) [inline src/uielements/message/message.component.js:26]: why is this `svg` hardcoded. User can pass aything

File: src/uielements/message/message.component.js [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+        </svg>
+      </span>
+    </div>
+  );
+}
+Message.PropTypes = {
```
Comment (udaypydi) [inline src/uielements/message/message.component.js:32]: Allow children as a prop. A message box can have an icon as well. It shouldn't be restricted to message text alone.

File: src/uielements/message/message.component.js [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+  /**
+   * Renders a button based on the primary theme
+   */
+  primary: PropTypes.bool,
```
Comment (udaypydi) [inline src/uielements/message/message.component.js:36]: remove primary in this case

**Comment by udaypydi** [comment]
@mvhk there are lint issues in the PR. pls fix them. and the commit messages are also not according to commit lint. Please add proper commit messages according to the convention

File: src/uielements/message/message.component.js (hunk 11 above)
Comment (mvhk) [reply src/uielements/message/message.component.js:36]: default? instead of primary

Comment (mvhk) [reply src/uielements/message/message.component.js:36]: removed

File: src/uielements/message/message.component.js (hunk 9 above)
Comment (mvhk) [reply src/uielements/message/message.component.js:26]: removed svg

File: src/uielements/message/message.component.js (hunk 8 above)
Comment (mvhk) [reply src/uielements/message/message.component.js:16]: done

File: src/uielements/message/message.component.js (hunk 10 above)
Comment (mvhk) [reply src/uielements/message/message.component.js:32]: Done
